Makefile

```diff
@@ -224,14 +227,23 @@ _run_setup:
 # Run the app (standard mode)
 run:
 	@echo "$(YELLOW)Running the app...$(RESET)"
-	@$(MAKE) -s _run_setup
+	@$(MAKE) -s _run_check
+	@poetry run uvicorn opendevin.server.listen:app --port $(BACKEND_PORT) &
+	@echo "$(YELLOW)Waiting for the app to start...$(RESET)"
+	@until nc -z localhost $(BACKEND_PORT); do sleep 0.1; done
+	@echo "$(GREEN)Application started successfully.$(RESET)"
```

Comment:
This change changes the behavior of `make run` to not start the frontend, right? If so, please revert back so that this PR doesn't change the behavior of `make run`.

Reply:
Yes. The Frontend server is unnecessary to run in production as it is already built and linked with the backend.

Reply:
Thanks for the explanation. As I requested above, please revert this back so this PR does not change the behavior of `make run`, since `make run`.

Reply:
Why should a end user run the front-end server?

Reply:
added new mode for end users